src/server/updater/updatechecker.cpp

```diff
@@ -66,7 +66,11 @@ void UpdateChecker::versionInfoReceived(UniqueId jobId) {
         SentryHandler::instance()->captureMessage(SentryLevel::Warning, "AbstractUpdater::checkUpdateAvailable", ss.str());
         LOG_ERROR(Log::instance()->getLogger(), ss.str().c_str());
     } else {
-        _versionInfo = getAppVersionJobPtr->getVersionInfo(_channel);
+        DistributionChannel channel = _channel;
+        // Check if there is a progressive update ongoing
+        if (channel == DistributionChannel::Prod && getAppVersionJobPtr->hasProdNext()) channel = DistributionChannel::Next;
```

Comment:
Suggestion: re-use the description of the PR as a comment, i.e., mentions that `hasProdNext()` is `true` if and only if the running application has been elected for an upgrade in the context of a progressive update.

Reply:
Done in 276ebd43f7d2629b12f4da00df41acf89a33ff4b